digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadAddressServiceSpec.scala

```diff
@@ -794,7 +794,7 @@ class RoadAddressServiceSpec extends FunSuite with Matchers{
         createRoadAddressLink(Sequences.nextViitePrimaryKeySeqValue, 5622953, o5622953Geom, 2825, 3, 0, 279, 321, SideCode.TowardsDigitizing, Anomaly.None, false, true) // end calibration point for testing
       )
 
-      val addresses = oldAddressLinks.map(roadAddressLinkToRoadAddress(false))
+      val addresses = oldAddressLinks.map(roadAddressLinkToRoadAddress(false)).map(i => i.copy(commonHistoryId = 123456))
```

Comment:
Why don't we use  the second parameter of `roadAddressLinkToRoadAddress` instead of map all the values after?